otwcode/otwarchive
# AO3-6628 Reorder font family names

## Issue

https://otwarchive.atlassian.net/browse/AO3-6628

## Purpose

Changes the order of default fonts.

## Testing Instructions

How can the Archive's QA team verify that this is working as you intended?

With this style rule, if you don’t have either Lucida Grande or Lucida Sans Unicode installed, but you do have GNU Unifont, GNU Unifont will be used for almost all of the text on the site.
GNU Unifont is a useful thing to have on your computer because it has a glyph for every codepoint in Unicode, but it is not a pretty font, nor a particularly legible one. It should only be used as a last resort.
It should be changed to 'Lucida Grande', 'Lucida Sans Unicode', Verdana, Helvetica, sans-serif, 'GNU Unifont' so GNU Unifont is only used for characters that don’t have a glyph in the preceding fonts.

If you have a Jira account with access, please update or comment on the issue
with any new or missing testing instructions instead.

## References

No.

## Credit

Claire Carden - she/her

If you have a Jira account, please include the same name in the "Full name"
field on your Jira profile, so we can assign you the issues you're working on.

*Please note that if you do not fill in this section, we will use your GitHub account name and
they/them pronouns.*
